src/main/io/gps.c

```diff
@@ -1955,8 +1956,8 @@ void onGpsNewData(void)
 
     lastTimeUs = timeUs;
 
-    if (!(STATE(GPS_FIX) && gpsSol.numSat >= GPS_MIN_SAT_COUNT)) {
-        // if we don't have a 3D fix and the minimum sats, don't give data to GPS rescue
+    if (!STATE(GPS_FIX)) {
```

Comment:
Yes that looks good to me.
The GPS Rescue code handles itself in the event of not receiving data anymore.
That code could be tightened up a bit as well, although it works, it does seem far too complex.